azurerm/resource_arm_virtual_machine.go

```diff
@@ -374,6 +374,11 @@ func resourceArmVirtualMachine() *schema.Resource {
 							Optional: true,
 							Default:  false,
 						},
+						"timezone": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							ValidateFunc: validateAzureTimeZone(),
+						},
```

Comment:
this also needs a state migration to update the `resourceArmVirtualMachineStorageOsProfileWindowsConfigHash` function and a default value (UTC)

Reply:
Sure. Is there any samples? I found [one here](https://github.com/terraform-providers/terraform-provider-azurerm/blob/83ecdac05b2479499a564988b4dbe2e3cd7706de/azurerm/resource_arm_storage_account_migration.go).

Reply:
that'll do as an example, there's a few in the codebase - this value also needs to be added to the Hash function, otherwise this won't be detected as ForceNew for existing Virtual Machines (which is why the state migrations needed)

Reply:
Added.